sdk/monitor/query/azlogs/autorest.md

```diff
@@ -4,7 +4,7 @@
 title: Logs Query Client
 clear-output-folder: false
 go: true
-input-file: https://github.com/Azure/azure-rest-api-specs/blob/0373f0edc4414fd402603fac51d0df93f1f70507/specification/operationalinsights/data-plane/Microsoft.OperationalInsights/stable/2022-10-27/OperationalInsights.json
+input-file: https://github.com/Azure/azure-rest-api-specs/blob/0b64ca7cbe3af8cd13228dfb783a16b8272b8be2/specification/operationalinsights/data-plane/Microsoft.OperationalInsights/stable/2022-10-27/OperationalInsights.json
```

Comment:
Is it expected that this didn't produce any changes?

Reply:
Yes, there shouldn't be any differences between the commits. Just updating to the latest commit for logs to match the other languages.